packages/asset-swapper/contracts/src/TwoHopSampler.sol

```diff
@@ -27,96 +27,100 @@ contract TwoHopSampler {
     using LibBytesV06 for bytes;
```

Comment:
Should be the same logic as before, just tweaked the inputs/outputs and did some light refactors.